app.test.js

```diff
@@ -24,3 +24,52 @@ describe('Add and Remove Testing', () => {
     expect(todolist.childElementCount).toBe(1);
   });
 });
+
+describe('clear all and edit Testing', () => {
+  document.body.innerHTML = `
+    <input id='toDoInput'>
+    <button id='addBtn'></button>
+    <div id='toDoListItemContainer'></div>
+    `;
+
```

Comment:
- [ ] Good job so far, kindly add a function for editing the task description as per the requirements.